daq/topology.py

```diff
@@ -429,11 +430,18 @@ def _generate_switch_local_acls(self, portset_acls, local_acl):
 
     def _generate_port_target_acls(self, portset_acls):
         port_set_mirrors = {}
-        for target in self._port_targets.values():
+        targets = list(self._port_targets.values())
+        for devices in self._set_devices.values():
+            for device in devices:
+                if device and device.gateway:
+                    target = {'port': None, 'port_set': device.gateway.port_set, 'mac': device.mac}
+                    targets.append(target)
+        for target in targets:
             port_no = target['port']
             port_set = target['port_set']
             target_mac = target['mac']
-            mirror_port = self.mirror_port(port_no)
+            # Mirror port is set to tap_intf, which is pri trunk port if no target port
```

Comment:
I don't understand this comment. It's not explaining "why" and/or it doesn't match what the code says.

Reply:
Added more detailed comment.